src/models/StyleSheet.js

```diff
@@ -275,13 +275,16 @@ export default class StyleSheet {
     const tag = this.getTagForId(id)
 
     /* add deferred rules for component */
-    if (this.deferred[id]) {
-      this.deferred[id].push(...cssRules)
-
-      // $FlowFixMe
-      tag.insertRules(id, this.deferred[id], name)
+    if (this.deferred[id] !== undefined) {
```

Comment:
Can this be one-lined?

Reply:
I suppose it could but ... readability 😅

Reply:
Yeah I was just thinking code size. But it's cool, LGTM